source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3412,7 +3412,7 @@ export class ChoiceSetInput extends Input {
     defaultValue?: string;
 
     @property(ChoiceSetInput.styleProperty)
-    style?: "compact" | "expanded" | "filtered";
+    style?: "compact" | "expanded" | "filtered" | "people";
```

Comment:
This change would mean that card authors attempting to use the `people` style outside of Teams would encounter undefined behavior since the `people` style is only supported by Teams.

If there's a correct fix to be done in this repo, it would likely be in the Designer portion of the codebase ([here](https://github.com/microsoft/AdaptiveCards/tree/main/source/nodejs/adaptivecards-designer)).

However, looking at your original bug again, I notice that the screenshot is not of our official Designer (at https://adaptivecards.io/designer/), but rather the Designer embedded in another experience. Where are you encountering this? The reason I ask is that the official version of the Designer does not have a "Save" facility. It's possible that whatever is hosting the Designer is what's responsible for you not being able to save...